pkg/antctl/raw/supportbundle/command.go

```diff
@@ -104,7 +105,8 @@ func init() {
 		Command.Flags().StringVarP(&option.dir, "dir", "d", "", "support bundles output dir, the path will be created if it doesn't exist")
 		Command.Flags().StringVarP(&option.labelSelector, "label-selector", "l", "", "selector (label query) to filter Nodes for agent bundles, supports '=', '==', and '!='.(e.g. -l key1=value1,key2=value2)")
 		Command.Flags().BoolVar(&option.controllerOnly, "controller-only", false, "only collect the support bundle of Antrea controller")
-		Command.Flags().StringVarP(&option.nodeListFile, "node-list-file", "f", "", "only collect the support bundle of Antrea controller")
+		Command.Flags().StringVarP(&option.nodeListFile, "node-list-file", "f", "", "only collect the support bundle of specific nodes filtered by names in a file (one node name per line)")
+		Command.Flags().Uint32Var(&option.days, "days", 0, "only collect logs that are generated since specific days ago")
```

Comment:
Thanks for the change! It looks good to me overall and we can add a new example with days filter in remoteControllerExample. Also, have you tested the function on the live setup?

Reply:
Thanks, I will add a new example in the remoteContrllerExample. I have tested it on a linux cluster. Fow windows node, i'm still trying to setup a working env.

Reply:
```suggestion
		Command.Flags().Uint32Var(&option.days, "days", 0, "if > 0, only collect logs that are generated since specific days ago")
```
Could we add the precondition to make it more clear?

Reply:
Sure

Reply:
Windows env is tested. Also:
1. add logrotate for FATAL log
2. add comm line example